homeassistant/components/backup/websocket.py

```diff
@@ -55,6 +55,7 @@ async def handle_info(
             "backups": list(backups.values()),
             "last_attempted_automatic_backup": manager.config.data.last_attempted_automatic_backup,
             "last_completed_automatic_backup": manager.config.data.last_completed_automatic_backup,
+            "last_completed_automatic_backup_id": manager.config.data.last_completed_automatic_backup_id,
```

Comment:
What is consuming this data?

Reply:
tbh, I added it because it felt right to do that and keep the data staructures in sync, but you're right, there is no consumer on the frontend side, yet ... so I'll remove it